recipe/meta.yaml

```diff
@@ -113,8 +106,9 @@ outputs:
         ###################################################
         # run-like
         - alembic >=1.13.0
-        - astropy >=6.0.0,<6.1.5
-        - astroquery >=0.4.6,<0.4.8
+        - astropy-base >=7.0.1, <8
+        - astroplan >=0.10.1
+        - astroquery ==0.4.9
```

Comment:
This pin is going to be really destructive in the future. Can we allow a range here?

Reply:
Astroquery tell me they reserve the right to break everything with a minor patch release (and are moving to arbitrary date numbers). They broke us last month going from 0.4.7 to 0.4.8. https://github.com/astropy/astroquery/issues/3185
The alternative is to let it float completely until it breaks and then set the upper limit.

Reply:
OMG ok. Hopefully they don't pin their deps too tightly. :)